ai_chatbots/chatbots.py

```diff
@@ -533,6 +536,39 @@ async def get_completion(
             log.exception("Error running AI agent")
 
 
+def get_problem_from_edx_block(edx_module_id, block_siblings):
+    """
+    Make an call to the learn contentfiles api to get the problem xml and problem
+    set xml using the block id
```

Comment:
Minor nitpick, could be added in a subsequent PR: Args/Returns sections in docstrings and/or variable/return types above.  Ditto for the new function below this one.

Reply:
done